devel/ansible/roles/datagrepper/files/datagrepper.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=datagrepper
+
+[Service]
+User=vagrant
+WorkingDirectory=/home/vagrant/datagrepper
+ExecStart=/bin/sh -c 'source /srv/venv/bin/activate && poetry run python devel/runserver.py --host "0.0.0.0" --port 5000'
```

Comment:
Using the `/srv/venv/bin/poetry` binary should be enough, no need to source `activate`.

Reply:
i get permissions errors when doing it this way (had the same issue witht he datanommer vagrant setup)

doing it this way works, is there an issue with doing it in this manner?

Reply:
No, it's fine. Maybe add a comment to explain it? I suppose the permission errors come from SELinux, a service file may not be allowed to run an executable in `/srv`. Anyway.